internal/builtinfunctions/functions.go

```diff
@@ -550,7 +550,10 @@ func getBindConstantsFunction() schema.CallableFunction {
 		schema.NewDisplayValue(
 			schema.PointerTo("Bind Constants"),
 			schema.PointerTo(
-				"Creates a list of objects with ID `CombinedObject`. "+
+				"Returns a list of objects each containing two properties: an item property which contains "+
+					"the corresponding item from the list in the first parameter; and, a constant property which "+
+					"contains the value of the second input parameter. "+
+					fmt.Sprintf("The two names are joined with the delimiter %q. ", CombinedObjIDDelimiter)+
 					fmt.Sprintf("Each object has two properties `%s` and `%s`.\n", CombinedObjPropertyItemName, CombinedObjPropertyConstantName)+
 					fmt.Sprintf("Param 1: Value(s) to be included in the `%s` field \n", CombinedObjPropertyItemName)+
 					fmt.Sprintf("Param 2: Value(s) to populate the field `%s` with every output item", CombinedObjPropertyConstantName)),
```

Comment:
So, the description starts with
> Returns a list of objects each containing two properties: an item property which [...] and, a constant property which [...]

and then, after a brief digression, it continues with
> Each object has two properties `%s` and `%s`

Those are somewhat redundant.  It would be good to combine their salient portions so that we don't have to repeat too much.

And, it would probably be better to move the "digression" about the schema to the end of the description, with suitable transition and framing words, so the reader will know what you're trying to describe.

Oh, and, by the way...I know you didn't do it in this PR, but newlines in this text get stripped when the text is rendered on the web site, so you might want to think a bit on how this text can or should be "formatted".  (E.g., I think you can use simple HTML tags, like `<br>` if you really want line breaks, but it might be best if you can present the information in a way which doesn't require explicit formatting.)

Reply:
has this been resolved?

Reply:
Yes; I just got interrupted before I could hit the button.  😆